ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -133,6 +150,30 @@ export class PrepareMigrationComponent implements OnInit {
     localStorage.removeItem(MigrationDetails.DataProgressMessage)
     localStorage.removeItem(MigrationDetails.SchemaProgressMessage)
   }
+  openConnectionProfileForm(isSource: boolean) {
+    let dialogRef = this.dialog.open(ConnectionProfileFormComponent, {
+      width: '30vw',
```

Comment:
Try to avoid and use common components defined in the styles/common CSS

Reply:
will move the CSS bits in follow up PR